lib/rucio/core/scope.py

```diff
@@ -79,7 +85,7 @@ def bulk_add_scopes(scopes, account, skipExisting=False, *, session: "Session"):
 
 
 @read_session
-def list_scopes(filter_: Optional[dict[str, Any]] = None, *, session: "Session") -> list[str]:
+def list_scopes(filter_: Optional[dict[str, Any]] = None, *, session: "Session") -> list[models.InternalScope]:
```

Comment:
`InternalScope` is defined in `types`, best to import it from there: https://github.com/rucio/rucio/blob/a1d68d02d7d634827236d1e052a32b9a0c6bf516/lib/rucio/common/types.py#L112

Reply:
Importing `InternalScope` from `from rucio.common.types` now